tests/test_crypto/test_fetchai.py

```diff
@@ -639,3 +644,62 @@ def test_execute_shell_command(mock_api_call):
 
     res = cosmos_api._execute_shell_command(["test", "command"])
     assert res == {"SOME": "RESULT"}
+
+
+def send_remaining_funds_back_to_faucet(account: FetchAICrypto):
```

Comment:
Ideally this is implemented as a decorator so we can decorate test functions with it.